requirements.txt

```diff
@@ -15,11 +15,7 @@ grpcio-health-checking>=1.62.0
 grpcio-reflection>=1.62.0
 grpcio-tools>=1.62.0
 
-# Pinning to older version of protobuf required for tensorflow==2.14.1
-protobuf>=4.25.3,<5.0
-
 # Open-telemetry logging
-opentelemetry-api>=1.23.0,<=1.27.0
-opentelemetry-sdk>=1.23.0,<=1.27.0
-opentelemetry-exporter-otlp>=1.23.0,<=1.27.0
```

Comment:
Removed the cap to `<=1.27.0`. That was to keep protobuf below 5.0, but `leaf-server-common` shouldn't really care.
For reference, `opentelemetry-exporter-otlp` is the one with the protobuf dependency.